docs/api/createApi.md

```diff
@@ -30,22 +30,23 @@ The main point where you will define a service to use in your application.
 ```ts title="Simulating axios-like interceptors with a custom base query"
 const baseQuery = fetchBaseQuery({ baseUrl: '/' });
 
-function baseQueryWithReauth(arg, api) {
-  let result = baseQuery(arg, api);
+const baseQueryWithReauth: BaseQueryFn = async (args, api, extraOptions) => {
```

Comment:
Shouldn't at least the `Args` generic argument for `BaseQueryFn` be specified in some way?

If this just wraps `baseQuery`, we should go with `BaseQueryFn<string | FetchArgs, unknown, FetchBaseQueryError, {}>`

Reply:
Good point, sorry I missed that 🤦

Reply:
I may go with `BaseQueryFn<string | FetchArgs, unknown, FetchBaseQueryError>`. Eslint is complaining on using {} as a type and It is inferred.

Reply:
Yeah, that's okay as well. 
Although I have to add that that eslint rule should be burned, since it's description is very misleading :disappointed: